src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -849,6 +849,28 @@ ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v8::Softmax>& nod
     return generate(std::dynamic_pointer_cast<ov::Node>(node), port, elemType, targetShape);
 }
 
+ov::runtime::Tensor generate(const
+                             std::shared_ptr<ov::op::v1::DeformablePSROIPooling>& node,
+                             size_t port,
+                             const ov::element::Type& elemType,
+                             const ov::Shape& targetShape) {
+    if (port == 1) {
+        ov::Tensor tensor(elemType, targetShape);
+        auto data_input_shape = node->input(0).get_shape();
+        const auto batch_distrib = data_input_shape[0] - 1;
+        const auto height = data_input_shape[2] / node->get_spatial_scale();
+        const auto width  = data_input_shape[3] / node->get_spatial_scale();
+
+        ov::test::utils::fill_data_roi<ov::element::f32>(tensor, batch_distrib, height, width, 1.0f, true);
+        return tensor;
+    } else if (port == 2) {
+        ov::Tensor tensor(elemType, targetShape);
+        ov::test::utils::fill_tensor_random(tensor, 1.8, -0.9);
```

Comment:
Could you please to use special structure (by default way) to avoid generation out of ranges in conformance cases?

Reply:
As we already discussed, default way does not allow to generate values in interval with real boundaries. 
As soon as ov::test::utils::create_and_fill_tensor is updated, we can change this code